config/tests/samples/create/harness.go

```diff
@@ -448,6 +448,10 @@ func MaybeSkip(t *testing.T, name string, resources []*unstructured.Unstructured
 			case schema.GroupKind{Group: "", Kind: "Secret"}:
 
 			case schema.GroupKind{Group: "serviceusage.cnrm.cloud.google.com", Kind: "Service"}:
+				if os.Getenv("GOLDEN_REQUEST_CHECKS") != "" {
+					// no golden log for this resource  yet
```

Comment:
Nit: Can we add a TODO here and create a ticket to capture logs later and remove this special handling?

Reply:
Yes currently in the process of running a capture for this.